tests/Fixer/FunctionNotation/SimpleLambdaCallFixerTest.php

```diff
@@ -150,5 +157,30 @@ public function provideFix70Cases()
                 call_user_func(static function ($a, $b) { var_dump($a, $b); }, 1, 2);
             ',
         ];
+        yield 'complex cases' => [
+            '<?php
+                call_user_func(\'a\'.$a.$b, 1, 2);
+                ($a/**/.$b)(1, 2);
+                (function (){})();
+                ($a["b"]{"c"}->a)(1, 2, 3, 4);
+                ($a::$b)(1, 2);
+                ($a[1]::$b[2]{3})([&$c], array(&$d));
+            ',
+            '<?php
+                call_user_func(\'a\'.$a.$b, 1, 2);
+                call_user_func($a/**/.$b, 1, 2);
+                \call_user_func(function (){});
+                call_user_func($a["b"]{"c"}->a, 1, 2, 3, 4);
+                call_user_func($a::$b, 1, 2);
+                call_user_func($a[1]::$b[2]{3}, [&$c], array(&$d));
+            ',
+        ];
+        yield [
```

Comment:
This `yield` is missing key.

Reply:
the key is optional and I can't think of a good one... : (